tests/integration/test_skytap.py

```diff
@@ -95,34 +82,6 @@ def assert_selected_option(self, menu, expected_value, expected_text):
         self.assertEqual(selected_option_value, expected_value)
         self.assertEqual(selected_option_text, expected_text)
 
-    def test_keyboard_layouts(self):
-        """
-        Test that menu for selecting keyboard layout defaults to appropriate value.
-
-        - When accessing Skytap XBlock instance for the first time,
-          menu should default to "English (US)".
-        - On subsequent visits, menu should default to keyboard layout that
-          was selected when learner last clicked button for launching exercise environment,
-          across different instances of the Skytap XBlock.
-        """
-        self.load_scenario("xml/skytap_multiple.xml")
-
-        menus = self.find_menus()
-        for menu in menus:
-            self.assert_selected_option(menu, "us", "English (US)")
```

Comment:
@afzaledx `assert_selected_option` is no longer used after this change, could you please remove it?